starter-code/css/main.css

```diff
@@ -6,3 +6,211 @@ Green: #00B172
 White: #FFF
 
 */
+/* ---- reset ---- */
+
+html {
+    box-sizing: border-box;
+  }
+  *, *:before, *:after {
+    box-sizing: inherit;
+  }
+  
+  
+  
```

Comment:
you have to reset the body too. If you inspect the body tag in your browser, you'll see that it has an ugly white thin margin. So you have to set body margin an padding to 0.